rspace/src/main/scala/coop/rchain/rspace/history/package.scala

```diff
@@ -81,10 +81,10 @@ package object history {
     }
 
   // TODO(ht): make this more efficient
-  private[rspace] def commonPrefix[A](a: Seq[A], b: Seq[A]): Seq[A] =
+  private[this] def commonPrefix[A](a: Seq[A], b: Seq[A]): Seq[A] =
     a.zip(b).takeWhile { case (l, r) => l == r }.map(_._1)
```

Comment:
We don't know much about type `A` here, so I think it makes more sense to do the comparison using `.equals`. `==` might work most of the time, but not always, so there is a chance of deferred errors

Reply:
Might be better to put an `Eq` constraint on `A`, and use cats' `===`.